### PR TITLE
feature - add flags to leaderboard

### PR DESCRIPTION
**Pull request by hydr063n**
##### Description

I've added functionality to load flag icon next to the country in Community leaderboard

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/4152

##### Testing

Before: https://user-images.githubusercontent.com/21199234/55767457-a3ba4a80-5a96-11e9-8a29-7b1da9e1e30e.png

After:
1. Personally curated test data 4 countries: https://imgur.com/a/8UQcuSj

2. If the mapping of Country name to flag fails the default flag is USA e.g Incorrectly spelled Yemen displays US flag https://imgur.com/a/qMRwD15

3. Real Rinkeby data https://imgur.com/a/KhC84UK

I opted for the flag API instead of emojis as (from my personal research) Microsoft Windows doesn't support flag emojis. Idk, works on my Linux but the userbase of Linux compared to Windows is small.

### CODE REVIEW

File: app/marketing/views.py [hunk 1]
```diff
@@ -790,6 +790,24 @@ def leaderboard(request, key=''):
             for tech in techs:
                 technologies.add(tech)
 
+    import pandas
```
Comment (danlipert) [inline app/marketing/views.py:793]: Can you move this import to the top of the file?

Comment (danlipert) [reply app/marketing/views.py:793]: Actually, we don't need to use pandas for this - you could either take that list of country codes and set it up as a python file and just import that here. I think thats easiest.

File: app/marketing/views.py [hunk 2]
```diff
@@ -845,8 +863,11 @@ def leaderboard(request, key=''):
     if next_update and next_update < timezone.now():
         next_update = timezone.now() + timezone.timedelta(days=1)
 
+    dualList= zip(items[0:limit],flags)
```
Comment (danlipert) [inline app/marketing/views.py:866]: please use snake case for variable names: https://en.wikipedia.org/wiki/Snake_case

File: app/marketing/views.py (hunk 1 above)
Comment (hydr063n) [reply app/marketing/views.py]: I've converted the csv into a python list and set the required flag variables inside the new file

File: app/marketing/views.py (hunk 2 above)
Comment (hydr063n) [reply app/marketing/views.py]: whoops. fixed!

**Pull request merged**